test/fixtures/hello-cloud/stateful_set.yml

```diff
@@ -11,9 +11,9 @@ spec:
         app: hello-cloud
         name: nginx-ss
     spec:
+      terminationGracePeriodSeconds: 0
```

Comment:
This cuts the time from ~12s to ~6s by SIGKILL the pods immediately instead of letting nginx shut down nicely. Note that k8s v1.6 does not have the field required to shut down all SS pods at once, so they're still being shut down in an ordered fashion. Interestingly, decreasing the replicas to 1 doesn't seem to help.